examples/proc_FID_nutation.py

```diff
@@ -0,0 +1,97 @@
+"""
+Process FID nutation data
+====================
+`py proc_FID_nutation.py NODENAME FILENAME EXP_TYPE`
+
+Fourier transforms (and any needed data corrections for older data) are performed according to the `postproc_type` attribute of the data node.
+This script plots the result as well as examines the phase variation along the indirect dimension.
+Finally the data is integrated and fit to a sin function to find the optimal beta_ninety.
+
+Tested with:
+
+``py proc_FID_nutation.py FID_nutation_1 240805_amp0p1_27mM_TEMPOL_FID_nutation.h5 ODNP_NMR_comp/nutation``
+"""
+import pyspecdata as psd
+import pyspecProcScripts as prscr
+import matplotlib.pyplot as plt
+import sympy as sp
+import numpy as np
+import sys
+
+signal_range = (-250, 250)
+assert len(sys.argv) == 4
+s = psd.find_file(
+    sys.argv[2],
+    exp_type=sys.argv[3],
+    expno=sys.argv[1],
+    lookup=prscr.lookup_table,
+)
+fig, (ax1, ax2, ax3) = plt.subplots(1, 3)
+fig.set_figwidth(15)
+fig.set_figheight(6)
+with psd.figlist_var() as fl:
+    # {{{ set up subplots
+    fl.next("Raw Data with averaged scans", fig=fig)
+    fig.suptitle("FID Nutation %s" % sys.argv[2])
+    # }}}
+    if "nScans" in s.dimlabels:
+        s.mean("nScans")
+    # {{{ Apply overall zeroth order correction
+    s = prscr.select_pathway(
+        s["t2":signal_range], s.get_prop("coherence_pathway")
+    )
+    s.ift("t2")
+    s /= prscr.zeroth_order_ph(s["t2":0])
+    s.ft("t2")
+    fl.image(s, ax=ax1)
+    ax1.set_title("Signal pathway / ph0")
+    # }}}
+    # {{{ Look at phase variation
+    d_integral = s.C.real.integrate("t2")
+    phase_ind_beta = (
+        s.C
+    )  # copy for applying zeroth order to individual indirect indices
+    for j in range(len(s.getaxis("beta"))):
+        phase_ind_beta["beta", j] /= prscr.zeroth_order_ph(
+            phase_ind_beta["beta", j]
+        )
+    phase_ind_beta.real.integrate("t2")
```

Comment:
This doesn't do anything, because the real property creates a copy. If you want to process the data in place, you can use the run function, which you can pass np.real.